.circleci/config.yml

```diff
@@ -49,6 +49,10 @@ jobs:
           if [[ "${CIRCLE_NODE_INDEX}" == 3 ]]; then
              make test-py37
           fi
+      - run: |
+          if [[ "${CIRCLE_NODE_INDEX}" == 3 ]]; then
```

Comment:
I think we would remove one of the Python versions from CI in case of adding 3.8.

Reply:
Discussed offline - circle ci has 4 runners hence hitting the limit. Removed this change.